config/target_score.txt

```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-4
+8
```

Comment:
CI is red! again, if you do not resolve it by next Wednesday I will decrease you mark with:

* -1 for red CI
* -1 for months hardcode